ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -553,7 +562,9 @@ function AuthProviderForm({
                                                                 type="text"
                                                                 id={`requiredAttributes[${index}].attributeValue`}
                                                                 value={attribute.attributeValue}
-                                                                onChange={onChange}
+                                                                onChange={(event, _value) =>
+                                                                    onChange(_value, event)
+                                                                }
```

Comment:
If we're using `_value` by passing it through, let's not prepend `_`

```suggestion
                                                                onChange={(event, value) =>
                                                                    onChange(value, event)
                                                                }
```